File: C/3209.c
#include <stdio.h>

int main(){

    int n, numStrips, totalOutlets = 0, numOutlets;

    scanf("%i", &n);

    for(int i = 0; i < n; i++){
        totalOutlets = 0;
        scanf("%i", &numStrips);

        for(int j = 0; j < numStrips; j++){
            scanf("%i", &numOutlets);
            totalOutlets += numOutlets;
        }

        printf("%i\n", totalOutlets - numStrips + 1);

    }

    return 0;
}

File: Python/3209.py
n = int(input())

for i in range(n):
    numStrips, numOutlets = input().split(" ", 1)
    numOutlets = list(map(int, numOutlets.split()))

    numAppliances = sum(numOutlets) - int(numStrips) + 1

    print(numAppliances)